vitest.config.ts

```diff
@@ -17,6 +17,7 @@ export default defineConfig({
   test: {
     pool: 'vmThreads',
     clearMocks: true,
+    mockReset: true,
```

Comment:
I noticed that spies were leaking between tests which made it very hard to test something in image suite
this made some other tests to breka in Itext which I adapted

Reply:
yeah this should be the default honestly.